client-participation/.dockerignore

```diff
@@ -1,20 +1,8 @@
-node_modules
-bower_components
-public
-.DS_Store
```

Comment:
Unsure on this; I think it's just a mac os filesystem thing, and I don't know how smart docker is about ignoring this by default. I'm less convinced it should be in the `.gitignore`, since mac devs should be responsible for having OS specific stuff like this in their global `~/.gitignore`. Then again, there's an argument that the project is more newby friendly if their are fewer of these gotchas, but I also don't want to overoptimize for that.

Reply:
Its pretty common to see `.DS_Store` in `.gitignore` files just like its pretty common to see IDE specific ignores e.g. `.vscode`. There is next-to-no harm there being these extra entries and it does save some accidental commits and accidently importing some bogus cache files from other people's computers etc. It's in the `polis-whitelabel` fork for instance.

Reply:
Also prefer to keep these! Happy to have them wherever (global gitignore perhaps not participationView for instance) but we should be ignoring these.

Reply:
bower_components is just gone, hopefully, speaking here about things like `.DS_Store`